erpnext/www/book_appointment/index.py

```diff
@@ -14,7 +14,8 @@ def get_context(context):
 	if is_enabled:
 		return context
 	else:
-		frappe.local.flags.redirect_location = '/404'
+		frappe.redirect_to_message(_("Appointment Scheduling Disabled"), _("Appointment Scheduling has been disabled for this site")
```

Comment:
Sider points to undefined `_`

You will have to add an import for the translation string
`from frappe import _`